apps/designer/app/shared/pubsub/index.ts

```diff
@@ -0,0 +1,9 @@
+import { createPubsub } from "@webstudio-is/react-sdk";
```

Comment:
This file is the core of the magic (together with the `createPublish` declaration in the SDK)